src/code/audio_heap.c

```diff
@@ -11,12 +11,12 @@ void AudioHeap_DiscardSampleBank(s32 sampleBankId);
 void AudioHeap_DiscardSampleBanks(void);
 
 /**
- * Effectively scales `updatesPerFrameInv` by the reciprocal of `scaleInv`
- * `updatesPerFrameInvScaled` is just `updatesPerFrameInv` scaled down by a factor of 256.0f
- * i.e. (256.0f * `updatesPerFrameInvScaled`) is just `updatesPerFrameInv`
+ * Effectively scales `ticksPerUpdateInv` by the reciprocal of `scaleInv`
+ * `ticksPerUpdateInvScaled` is just `ticksPerUpdateInv` scaled down by a factor of 256.0f
+ * i.e. (256.0f * `ticksPerUpdateInvScaled`) is just `ticksPerUpdateInv`
  */
 f32 AudioHeap_CalculateAdsrDecay(f32 scaleInv) {
-    return (256.0f * gAudioCtx.audioBufferParameters.updatesPerFrameInvScaled) / scaleInv;
+    return (256.0f * gAudioCtx.audioBufferParameters.ticksPerUpdateInvScaled) / scaleInv;
```

Comment:
I think the comment can just go tbh
It's more confusing than useful

Reply:
I still kept the first line of it, just left out the explanation on why the 256.0f is effectively canceled out